tick/u.q

```diff
@@ -3,16 +3,16 @@
 /2006.05.08 add
 
 \d .u
-init:{w::t!(count t::tables`.)#()}
+init:{(w;t)::(t!();tables`.)}
 
-del:{w[x]_:w[x;;0]?y};.z.pc:{del[;x]each t};
+(del;.z.pc):({w[x]_:w[x;;0]?y};{del[;x]each t})
 
-sel:{$[`~y;x;select from x where sym in y]}
+sel:{(select from x where sym in y;x)`~y}
 
-pub:{[t;x]{[t;x;w]if[count x:sel[x]w 1;(neg first w)(`upd;t;x)]}[t;x]each w t}
+pub:{[t;x]{[t;x:sel[;w 1];w]if[count x;(neg first w)(`upd;t;x)]}[t;x]each w t}
 
 add:{$[(count w x)>i:w[x;;0]?.z.w;.[`.u.w;(x;i;1);union;y];w[x],:enlist(.z.w;y)];(x;$[99=type v:value x;sel[v]y;@[0#v;`sym;`g#]])}
 
-sub:{if[x~`;:sub[;y]each t];if[not x in t;'x];del[x].z.w;add[x;y]}
+sub:{[x:{$[not x in t,`;'x;x]};y]if[x~`;:sub[;y]each t];del[x].z.w;add[x;y]}
```

Comment:
What is the purpose to add
```
,`
```
at the end of the condition?

Reply:
From what I have been able to guess, since the ifs' are switched, you need to check if `x` is the null symbol on the first condition as well, and this can be achieved using `in`, like it has been done here. Still, I don't really see the point of doing this unless we get a speed benefit of some sort.

Reply:
Well, I just added it because I tried to incorporate the new syntax as much as possible. I need to measure how fast or slow this is, so once I do that, I'll determine if it's worthwhile or not